src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -319,12 +319,16 @@ int __ESBMC_rounding_mode = 0;
 
 void *__ESBMC_memset(void *, int, unsigned int);
 
+void __ESBMC_finish_formula();
```

Comment:
this is probably not needed.

Reply:
I am not against having an intrinsic that makes the symex stop the current path. There is a use case that is different from `assume(0)`

```c
// some code
assume(0);
while(1)
 // some code
```

Is different from having an intrinsic that just stop the analysis. But we can move this to another PR